Tutorials/Tutorial27_PostProcessing/assets/shaders/GenerateGeometry.fx

```diff
@@ -0,0 +1,233 @@
+#include "BasicStructures.fxh"
```

Comment:
Let's rename this file to Scene to be consistent with other tutorials

Reply:
Other files contain a verb. Maybe the file should be named `GenerateScene.fx`?